apps/guardian-ui/src/admin/FederationAdmin.tsx

```diff
@@ -34,15 +34,18 @@ export const FederationAdmin: React.FC = () => {
 
   useEffect(() => {
     if (!inviteCode) return;
-    api.config(inviteCode).then(setConfig).catch(console.error);
+    api.config().then(setConfig).catch(console.error);
```

Comment:
It's a little weird that we can just fetch the config without an invite code cc @joschisan @elsirion. I understand we can update this behavior in the future but doesn't seem quite right to me.